optimum/exporters/openvino/__main__.py

```diff
@@ -361,6 +391,7 @@ class StoreAttr(object):
         preprocessors=preprocessors,
         device=device,
         trust_remote_code=trust_remote_code,
+        patch_16bit_model=patch_16bit,
```

Comment:
could we instead update and use `ov_config.dtype` instead of adding `patch_16bit_model` ?

Reply:
layers patching is time consuming process we would like to avoid doing that for cases where it is not really neseccary. We need it only for pytorch model wegiths represented in fp16 and bf16 format for avoid issues tracing on cpu (torchscript does not support some ops running on cpu in this format). ov_config.dtype currently used for output model type and signal to compress weights to fp16 after conversion, mixing them in 1 parameter may lead to some issues:

* inability to convert fp32 pytorch models to fp16 openvino model (now it is possible)
* compressing model weights twice (having model in fp16 on torch side and then repeat compression for openvino)